controller/telemetry/server.go

```diff
@@ -337,22 +337,22 @@ func (s *server) getNameAndDeployment(ip *common.IPAddress) (string, string) {
 	ipStr := util.IPToString(ip)
 	pods, err := s.pods.GetPodsByIndex(ipStr)
 	if err != nil {
-		log.Debugf("Cannot get pod for IP %s: %s", ipStr, err)
+		log.Errorf("Cannot get pod for IP %s: %s", ipStr, err)
```

Comment:
Can I bug you to revert the log level changes in this file? Most of these logs are not indicative of a larger problem. For instance, any external request will have a source IP that does not belong to a pod, so with this change we'd end up error logging on every request served.